config/routes.rb

```diff
@@ -1,10 +1,11 @@
-Rails.application.routes.draw do
-  get 'works/index'
-  root to: "works#index"
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-# root to: 'works#path'
-resources :works
-
+# frozen_string_literal: true
 
+Rails.application.routes.draw do
+  resources :works
+  resources :users, only: %i[index show]
```

Comment:
Interesting syntax... This is fine, but pretty inconsistent with the rest of the file and pretty difficult to read in this case; it may make more sense to just simply put `[:index, :show]`